src/Files.App/Utils/Cloud/Detector/GoogleDriveCloudDetector.cs

```diff
@@ -56,7 +66,10 @@ await FilesystemTasks.Wrap(() => StorageFile.GetFileFromPathAsync(Path.Combine(a
 				var folder = await StorageFolder.GetFolderFromPathAsync(path);
 				string title = reader["title"]?.ToString() ?? folder.Name;
 
-				App.AppModel.GoogleDrivePath = path;
+#if DEBUG
+				Debug.WriteLine("YIELD RETURNING from `GoogleDriveCloudDetector.GetProviders()` (roots): ");
+				Debug.WriteLine($"Name: Google Drive ({title}); SyncFolder: {path}");
+#endif
```

Comment:
Do we still need to output this data?

Reply:
I can remove it, I just had it in there to help with testing. Should I remove all of the `Debug.WriteLine` calls and their related code?

Reply:
It wouldn't hurt to keep it, but I don't think we need to wrap it in the `#if DEBUG` statement.

Reply:
Sure I can remove the `#if debug` statement there. Although @0x5bfa told me to add those in his review, just to let you know. Maybe he was thinking of the potential performance hit the `Debug.WriteLine` calls could cause in release.

There are 7 `#if debug`s in GoogleDriveCloudDetector and one in StorageDevicesService. Do you want me to remove all of them?

Reply:
I think we should.
On release builds, those lines shouldn't be run and you can't read unless you debug it via VS or WinDbg without source code.

Reply:
That would be helpful, thanks!

Reply:
No problem, I just pushed the change.

Reply:
I meant we should keep they as are.